Dockerfile

```diff
@@ -0,0 +1,18 @@
+#Using the official ubuntu OS as a Base Image
+FROM ubuntu:16.04
+
+USER root
+
+#Configuration
+RUN pip install -r requirements/base.txt
+RUN paver install_pages
+
+#Setting up environment variables. Change the value according to the value of the ENV_VAR
+
+ENV BASIC_AUTH_USER value	
+ENV BASIC_AUTH_PASSWORD value
+ENV USER_LOGIN_EMAIL value
+ENV USER_LOGIN_PASSWORD value
+
+#Edit the CMD command to run speciifc tests
```

Comment:
If we use the CMD directive to run the tests, it will require a user to rebuild the image every time they want to run a different paver command, i.e running a specific test/test suite. I think it would be better to have the ENTRYPOINT/CMD just be something like /bin/bash. You could run the container with the '-t' and '-d' flags to have it run detached, and then shell into the container to run arbitrary paver commands